module/item/item-sheet.js

```diff
@@ -319,6 +365,23 @@ export async function useItem(event){
         const item = actor.items.get(itemId);
         await item.update({"system.quantity": Number(duplicate(item.system.quantity)) - 1});
     }
+
+	// if the item is a food buff
+	if (game.ptu.data.items.find(i => i.name.toLowerCase().includes(itemName.toLowerCase())).system.consumable == 2){
+		//if there is a target actor add the food buff to the target actor, else add foodbuff to the user
+		
+		const effectedActor = targetedActor ? targetedActor : actor;
+		//if the target actor already has a food buff, add the new food buff to the list
+		if (effectedActor.system.digestionBuff.trim() != "" && effectedActor.system.digestionBuff.trim().toLowerCase() != "none"){
+			const buffArray = effectedActor.system.digestionBuff.split(", ");
+			buffArray.push(itemName);
+			const newBuffString = buffArray.join(", ");
+			await effectedActor.update({"data.digestionBuff": newBuffString});
+		}
+		else{
+			await effectedActor.update({"data.digestionBuff": itemName});
+		}		
+	}
```

Comment:
This method should be refactored slightly.

We can use `const item = actor.items.get(itemId);` outside of the scope of whether or not it's a pokemon, to get the item detail.

That way we won't have to go looking through all items to find whether it's consumable; we can just check the `item.system.consumable == 2`.
This '2' should also be turned into a constant. `CONFIG.PTUItems = {Consumable: {NOT: 0, YES: 1, FOODBUFF: 2}}` or the like.

Reply:
I see why you need to fetch it from the `game.ptu.data.items` list because it would be stored as plain text not as an item. Ignore that part of the comment.